packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/nodejs/summaryofsteps.md

```diff
@@ -1 +1,19 @@
-## Summary of step
\ No newline at end of file
+### Sequence flow (part 1)
+
+<div class="common-image-format">
+
+![Sign in with email and phone 1](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-nodejs-seq-1.png
+ "Sign in with email and phone 1")
+
+</div>
+
+### Sequence flow (part 2)
+
+<div class="common-image-format">
+
+![Sign in with email and phone 2](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-nodejs-seq-2.png
+ "Sign in with email and phone 2")
+
+</div>
+
+>**NOTE:** The SDK sample application does not support this use case.
```

Comment:
Can you be specific and indicate that the Node.js sample app doesn't support this use case (or maybe have a link to the sample app?).

Reply:
Thanks, removed this note as it was in error